sass/components/_page.scss

```diff
@@ -0,0 +1,22 @@
+$font-family: 'Roboto', 'sans-serif';
+
+.page {
+  font-family: $font-family;
+  &__body {
+    margin: 0;
+  }
+  &__body--no-scroll {
+    overflow: hidden;
+  }
+}
+
+.section__title {
```

Comment:
nesting - in order to use it, you need to declare a name of the block, even if it has no rules
```
.section {
  &__title {
    /* ... */
  }
}
```